### ISSUE TITLE
improve csv import

### ISSUE THREAD

**Issue by dmnfarrell** [dmnfarrell/pandastable#9]
improve import dialog to handle unusual formats
fix column header issues in preview

**Comment by dacoex** [comment]
Do you plan to support dateparsers?

**Comment by dmnfarrell** [comment]
Sure. I am happy to add whatever functionality is needed when I get the time. Though date parsers require the user can add their own custom function so I would have to figure out how to do that in the GUI.

**Comment by rswgnu** [comment]
Need to handle numeric conversion of string fields such as "$220.40" and "25%".  This and DataExplore are great concepts but they fail to handle commonly encountered inputs.  I would recommend you test with a bunch of the large csv sample files available across the Internet to quickly find any incompatibilities.

**Comment by dmnfarrell** [comment]
@rswgnu Yes this needs to be addressed but unfortunately this is a one man project and I haven't had time. Will try to look into it soon.

**Comment by rswgnu** [comment]
Understood.  Very impressive work for one person.  I just hope to be able to use it at some point.

-- Bob
